### DISCUSSION TITLE
検索画面がまた変わった

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#427] [要望・アイデア・不具合]

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご不便をおかけして、申し訳ありません。

大きな機能のアップデートではβ版をリリースしていますが、
軽微なものや影響範囲の小さいと考えられるものは最初からGeneral Availabilityとして提供をしています。
また、リリースまでの時間を短縮するため、実装できるものから段階的にリリースを行うこともあります。

今回の検索周りの改善は、影響範囲が検索ページに閉じているためβ版を挟まずにリリースを行っておりました。
予定していた検索周りの改善が落ち着いたため、リリースノートでの告知も行いました。
https://qiita.com/release-notes#2023-01-31

変更が小さい場合は、リリースノートでの告知をしない場合もあります。
ご希望に沿えないこともあるかと思いますが、ご理解いただけると幸いです🙇
